tests/fluka/test_fluka_converter.py

```diff
@@ -67,15 +69,22 @@ def parser() -> Parser:
     """Parser fixture."""
     return get_parser_from_str("fluka")
 
+@pytest.fixture
```

Comment:
As for now I'd suggest to use the same JSON file as shieldhit,
take a look at TOPAS:
https://github.com/yaptide/converter/blob/114-parsing-of-the-projectile-kinetic-energy-fluka/tests/topas/conftest.py

Please define conftest.py in fluka tests suite. There define a fixture which provides Fluka JSON dict by passing a shieldhit dict.

At a later stage we will replace it with proper Fluka JSON (once the project will differentiate by editor)